tests/rptest/tests/rbac_upgrade_test.py

```diff
@@ -76,5 +79,7 @@ def default_role_created():
             err_msg="Timeout waiting for default role to be created")
 
         # Verify that we don't get a license nag for the default role
+        # NOTE: This assertion will FAIL if running in FIPS mode because
+        # being in FIPS mode will trigger the license nag
```

Comment:
so these were failing only some of the time? or is it that fips mode tests are not executed on every run? or something else changed?

Basically I don't understand why this wasn't producing a huge number of red builds.

Reply:
The tests only run in fips mode nightly: https://buildkite.com/redpanda/vtools/settings/schedules/5dd6979c-5398-40b9-801d-f72ff0f6bb9f

That being said, strangely, I can't find either a failing or succeeding run of these three test in any of the recent runs. I'm wondering if that's because of ducktape shortcutting running these due to other test failures, pandatriage having marked them flaky disables them for the nightly fips run, or maybe something else.

Reply:
Weird. I seem to recall a situation where fips mode tests were not running in CDT or something, but I don't recall the details. Presumably nothing to do on this PR, but worth looking into @michael-redpanda?

Reply:
I'm chasing this but I'll merge the PR in the meantime.